esignet-service/src/test/java/io/mosip/esignet/controllers/LinkedAuthorizationControllerTest.java

```diff
@@ -410,12 +400,13 @@ public void authenticate_withInvalidChallengeList_thenFail() throws Exception {
                         .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn();
 
-        List<String> errorCodes = Arrays.asList(INVALID_AUTH_FACTOR_TYPE, INVALID_CHALLENGE, INVALID_CHALLENGE_FORMAT);
+        List<String> errorCodes = Arrays.asList(INVALID_CHALLENGE_FORMAT,INVALID_AUTH_FACTOR_TYPE, INVALID_CHALLENGE_FORMAT_FOR_AUTH_FACTOR_TYPE,INVALID_CHALLENGE);
```

Comment:
Here also INVALID_CHALLENGE should be in the error list as challenge is set to null.